src/status_im/extensions/core.cljs

```diff
@@ -207,6 +209,28 @@
                            {:failure-event-creator on-failure})
                          {:timeout-ms 5000})}))
 
+(defn- parse-ipfs-add-response [res]
+  (-> (parse-json res)
+      (select-keys [:Name :Hash :Size])
+      (set/rename-keys {:Name :name :Hash :hash :Size :size})))
+
+(re-frame/reg-event-fx
+ :ipfs/add
+ (fn [_ [_ _ {:keys [value on-success on-failure]}]]
+   (let [formdata (doto
+                   (js/FormData.)
+                    (.append constants/ipfs-add-param-name value))]
+     {:http-post (merge {:url  constants/ipfs-add-url
```

Comment:
Does this work fine with this one? Don't we need to introduce a raw version?

Reply:
no, it works nicely with the non-raw version! :)

Reply:
btw, I didn't add `:http-raw-post`, so `http/post` event still doesn't work..

Reply:
👍